common/config/babel/features.js

```diff
@@ -21,6 +21,8 @@ module.exports = {
     'meeting-id',
     // Demo feature. Used in live-documentation of conditional compilation. Do not use in production code.
     "in-progress-beta-feature-demo",
+    // Feature for Rich Text Editor (RTE) composite support and uploading images
+    "rich-text-editor-composite-and-image-upload"
```

Comment:
Just wondering if we should have 2 feature flags instead of 1 as image upload functionality will be inside components too (including public api)

Reply:
> Just wondering if we should have 2 feature flags instead of 1 as image upload functionality will be inside components too (including public api)

Unless we're anticipating that these 2 features will be released separately, I don't think we should add the complexity of having 2 feature flags. Instead, maybe I can rename this feature flag to `rich-text-editor-image-upload-and-composite-support` if that helps?